modules/customize-certificates-add-service-serving.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-{context}"]
+= Add a service certificate
+
+To secure communication to your service, have the cluster generate a 
+signed serving certificate and key pair into a secret in your namespace. To 
+do this, set the `service.beta.openshift.io/serving-cert-secret-name` 
+annotation on your service with the value set to the name you want to use 
+for your secret.
+
+.Procedure
+
+* Annotate the service with `service.beta.openshift.io/serving-cert-secret-name`.
```

Comment:
I am confused. Should this step be 
```
* Set the name of the secret that your service uses:
+
----
$ oc annotate service <service-name> service.beta.openshift.io/serving-cert-secret-name=<service-name> <1>
----
<1> For `<service-name>`, specify the name of the secret for your service.
...
```

Is there a good way to validate that this worked?

Reply:
I've annotated this command, and clarified what each of these replaceables mean. I've also added a way of describing the service to view the annotations.